features/other_a/reading.feature

```diff
@@ -78,7 +79,7 @@ Feature: Reading count
       And I follow "First work"
       And I am on testuser's works page
       And I follow "second work"
-      And I am on testuser2's works page
+      And I am on testuser2 works page
```

Comment:
Why remove the possessive?

Reply:
The uneven number of single quotes in this file is screwing up the syntax highlighting. Currently after this line, it's all blue like a quoted string that never ends:

https://github.com/otwcode/otwarchive/blob/17307a8a6294d2af7fab882ea5aa14eb786d51af/features/other_a/reading.feature#L81

Reply:
@ god why